pkgs/development/libraries/qt-6/hooks/qtbase-setup-hook.sh

```diff
@@ -26,6 +29,57 @@ else
     cmakeBuildType="Release"
 fi
 
+# Build tools are often confused if QMAKE is unset.
+QMAKE=@dev@/bin/qmake
+export QMAKE
+
+QMAKEPATH=
+export QMAKEPATH
+
+QMAKEMODULES=
+export QMAKEMODULES
```

Comment:
```suggestion
export QMAKE=@dev@/bin/qmake
export QMAKEPATH=
export QMAKEMODULES=
```
Why are we not doing this?

Reply:
Copied from qt5 hook with minimal changes, don't know why.

Reply:
Looks like shellcheck prefers separate export/assignment in some cases

Reply:
Thats only for subshells

Reply:
Changed it over for 6, didn't change the original file